shelley/chain-and-ledger/executable-spec/test/Generator/Core.hs

```diff
@@ -24,6 +24,7 @@ module Generator.Core
 
 import           Cardano.Crypto.VRF (deriveVerKeyVRF, genKeyVRF)
 import           Crypto.Random (drgNewTest, withDRG)
+import qualified Data.Map.Strict as Map
```

Comment:
better add explicit imports

Reply:
👍

Reply:
FWIW `Map` is usually used in this way. Otherwise you have to hide some operations from `Prelude`.